test/sequential/test-watch-mode.mjs

```diff
@@ -280,4 +280,30 @@ describe('watch mode', { concurrency: false, timeout: 60_000 }, () => {
 
     await failWriteSucceed({ file: dependant, watchedFile: dependency });
   });
+
+  it('should preserve output when --watch-preserve-output flag is passed', async () => {
+    const file = createTmpFile();
+    console.log(file);
```

Comment:
```suggestion
```
To keep consistency with other tests, there's no need to log the temporary file.

Reply:
Fixed